automatic/360ts/360ts.nuspec

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>360ts</id>
+    <version>9.0.0.1146</version>
+    <title>360 Total Security</title>
+    <authors>Qihu 360 Software</authors>
+    <owners>Chocolatey</owners>
```

Comment:
please change chocolatey owner to lowercase.
the original owner should also be present (comma seperated after chocolatey user).